test/src/unittests/rhythm/test_loopbpmconfidence.py

```diff
@@ -128,25 +131,27 @@ def testEmpty(self):
         self.assertEquals(0, confidence)
 
     def testZero(self):
-        zeroAudio = zeros(1000)
+        audio = MonoLoader(filename=join(testdata.audio_dir, 'recorded', 'techno_loop.wav'))() 
+        zeroAudio = zeros(len(audio))
         bpmEstimate = 0
         confidence = LoopBpmConfidence()(zeroAudio, bpmEstimate)
         self.assertEquals(0, confidence)
 
         # Non-zero estimate check results in non-zero confidence
-        # FIX-ME: Maybe given contant zero input, we should expect zero confidence
+        # FIXME: Maybe given contant zero input, we should expect zero confidence
         bpmEstimate = 125
         confidence = LoopBpmConfidence()(zeroAudio, bpmEstimate)
         self.assertNotEquals(0, confidence)
          
     def testConstantInput(self):
-        onesAudio = ones(100)
+        audio = MonoLoader(filename=join(testdata.audio_dir, 'recorded', 'techno_loop.wav'))() 
+        onesAudio = ones(len(audio))
         bpmEstimate = 0
         confidence = LoopBpmConfidence()(onesAudio, bpmEstimate)
         self.assertEquals(0, confidence)
 
         # Non-zero estimate check results in non-zero confidence
-        # FIX-ME: Maybe given contant input, we should expect zero confidence
+        # FIXME: Maybe given contant input, we should expect zero confidence
```

Comment:
as @ffont mentioned, this happens because the estimation is based on audio length. Therefore, different constant input length will result in different estimations and it is part of normal behavior. We can append this explanation to the comments here and remove the FIXME.